wagtail/core/templates/wagtailcore/shared/richtext.html

```diff
@@ -0,0 +1 @@
+<div class="rich-text">{{ html|safe }} LALA</div>
```

Comment:
I love your test content :)

` LALA` makes the tests fail.

Other than that, the approach here is 🔥 for not adding yet another setting.